app-ng/src/utilities.lisp

```diff
@@ -114,3 +110,15 @@ Version ~A is available from https://www.rigetti.com/forest~%"
        (format stream "~F+~Fi" re im))
       (t                                ; equiv: (minusp im)
        (format stream "~F-~Fi" re (- im))))))
+
+(defun generalized-boolean-to-exit-code (successp)
+  (cond ((integerp successp) successp)
+        ((null successp) 1)
```

Comment:
I learned about `qvm:boolean-bit` yesterday and it seems like a good fit here.

Reply:
Cool, thanks for the tip.